app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -15,7 +15,7 @@
     <%= render TextBlockWithTwoImagesComponent.new(
       heading: "Making maths add up",
       colour: "pink",
-      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_path: "static/images/content/hero-images/classroom-2.jpg",
       image_1_alt: "",
       image_2_path: "static/images/content/hero-images/0024.jpg",
```

Comment:
This image also needs replacing (could replace with general pic rather than maths one)?